src/components/rockets/RocketsFalcon.module.css

```diff
@@ -0,0 +1,4 @@
+.missionTable {
+  width: 90%;
+  margin: 5% auto;
```

Comment:
Hi, please check the css classname for the rockets section.

Reply:
Thx for the comments I will implement that in min.